db/options.xml.in

```diff
@@ -414,4 +415,54 @@
 			</control>
 		</subgroup>
 	</group>
+<!-- Picture-by-Picture ("split screen") and Picture-in-Picture settings -->
+	<group name="PIP/PBP">
+		<subgroup name="PIP/PBP settings">
+			<control id="PbP" type="list" name="PbP/PIP mode"  address="0xe9">
+				<value id="Off" name="Off"/>
+				<value id="PbP"  name="PbP"/>
+				<value id="PiP large"  name="PiP Large"/>
+				<value id="PiP small"  name="PiP Small"/>
+			</control>
+			<control id="maininputsource" type="list" name="Input Source Select (Main)">
```

Comment:
Duplicates `<control id="inputsource" ...` . Is it necessary?

Reply:
The problem lies in the `config.xml`: I would like to capture the allowed values for input sources only once and reuse them for PBP/PIP settings but they are bound within a group tag (were I do not want to place them - the group name "others" is definitely the last group I would search for the input source). And for a user to understand that the sub input source is in the PBP/PIP (sub) group but to set the input source for one of the split screens is in the "others" group under "input settings" is difficult to understand.

I will follow your usability proposals anyhow, please just suggest what you want and I push a new commit.